src/app/page.tsx

```diff
@@ -1,95 +1,20 @@
 import Image from "next/image";
 import styles from "./page.module.css";
+import InputText from "~/ui/InputText";
+import BtnAdd from "~/ui/BtnAdd/component";
```

Comment:
why is this line different than the one above? this should be `~/ui/BtnAdd` and that should work

Reply:
ready!